robottelo/cli/import_.py

```diff
@@ -36,13 +36,49 @@
     user                          Import Users (from spacewalk-report users).
 
 """
-from robottelo import ssh
+import os
+from robottelo import manifests, ssh
 from robottelo.cli.base import Base
 
 
 class Import(Base):
     """Imports configurations from another Satellite instances."""
     command_base = 'import'
+    repos = {
```

Comment:
I agree with @rplevka.  Standardizing these constants should be a separate effort.  They are not really in a standard format to be consumed by all tests. They are built in format for specific tests.